libs/sdk-ui-ext/src/internal/components/configurationControls/DataPointsControl.tsx

```diff
@@ -0,0 +1,45 @@
+// (C) 2019 GoodData Corporation
```

Comment:
2020 :)

Reply:
I guess the date updating script doesn't work on new files :-D